Competition2018/src/test/java/competition/subsystems/elevator/commands/MaintainerCommandTest.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.elevator.commands;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import competition.BaseCompetitionTest;
+import competition.subsystems.elevator.ElevatorSubsystem;
+import xbot.common.controls.actuators.mock_adapters.MockCANTalon;
+
+public class MaintainerCommandTest extends BaseCompetitionTest {
+    
+    ElevatorSubsystem elevator;
+    MaintainerCommand command;
+    
+    @Override
+    public void setUp() {
+        super.setUp();
+        
+        command = injector.getInstance(MaintainerCommand.class);
+        elevator = injector.getInstance(ElevatorSubsystem.class);
+        elevator.temporaryHack();
+        
+    }
+    
+    @Test
+    public void testSimple() {
+        command.initialize();
+        command.execute();
+    }
+    
+    @Test
+    public void checkGoUp() {
+        ((MockCANTalon)elevator.motor).setPosition(1200);
```

Comment:
Your test says `checkGoUp` but then you move the elevator up very high (and I'm assuming the target height is 0), and then run the command. That sounds like `checkGoDown` to me. =]

You should explicitly set the target height for these tests.

Reply:
done